templates/frontOffice/default/assets/src/js/thelia.js

```diff
@@ -474,7 +474,7 @@ var pseManager = (function($){
                 $.ajax({type: "POST", data: $(this).serialize(), url: url_action,
                     success: function(data){
                         $(".cart-container").html($(data).html());
-                        $.ajax({url:"ajax/addCartMessage", data:{ product_id: product_id, pse_id: pse_id },
+                        $.ajax({url:"/ajax/addCartMessage", data:{ product_id: product_id, pse_id: pse_id },
                             success: function (data) {
```

Comment:
This is not working is Thelia is installed in a subdirectory (e.g. http://www.yourhost.com/shop).

You have to use `{url path="/ajax/addCartMessage"}` to get the right URL. Às thelia.js is not parsed by Smarty, you have to find a way to pass this URL, either as a form parameter, or form data attribute, or something.